src/libs/actions/Link.js

```diff
@@ -34,33 +34,35 @@ function showGrowlIfOffline() {
 }
 
 /**
- * @param {String} url
+ * @param {String} [url] the url path
+ * @param {String} [shortLivedAuthToken]
+ *
+ * @returns {Promise<string>}
  */
-function openOldDotLink(url) {
-    /**
-     * @param {String} [shortLivedAuthToken]
-     * @returns {Promise<string>}
-     */
-    function buildOldDotURL(shortLivedAuthToken) {
-        const hasHashParams = url.indexOf('#') !== -1;
-        const hasURLParams = url.indexOf('?') !== -1;
+function buildOldDotURL(url, shortLivedAuthToken) {
+    const hasHashParams = url.indexOf('#') !== -1;
+    const hasURLParams = url.indexOf('?') !== -1;
```

Comment:
@akinwale @eVoloshchak can you remind me please why is this change here?

Reply:
To summarise: Some of the menu items made use of the `openOldDotLink(url)` method, which built the URLs using the inner defined `buildOldDotURL` method. Since the original issue scope was to make the "Copy to clipboard" behaviour happen for all links, I had to refactor this method to the outer scope.

You can follow the discussion here for more details: https://github.com/Expensify/App/pull/17310#issuecomment-1505594964

Reply:
@mountiny All done!